diff --git a/third_party/google/protobuf/BUILD b/third_party/google/protobuf/BUILD
index 0941d8c..d73f25a 100644
--- a/third_party/google/protobuf/BUILD
+++ b/third_party/google/protobuf/BUILD
@@ -581,7 +581,7 @@ cc_binary(
     linkstatic = 1,
     deps = select({
         "//conditions:default": [],
-        ":use_fast_cpp_protos": ["//external:python_headers"],
+        ":use_fast_cpp_protos": ["//third_party/google/protobuf/util/python:python_headers"],
     }),
 )
 
@@ -607,7 +607,7 @@ cc_binary(
         ":protobuf",
     ] + select({
         "//conditions:default": [],
-        ":use_fast_cpp_protos": ["//external:python_headers"],
+        ":use_fast_cpp_protos": ["//third_party/google/protobuf/util/python:python_headers"],
     }),
 )
 
@@ -656,7 +656,6 @@ py_proto_library(
     protoc = ":protoc",
     py_libs = [
         ":python_srcs",
-        "//external:six"
     ],
     srcs_version = "PY2AND3",
     visibility = ["//visibility:public"],
diff --git a/third_party/google/protobuf/protobuf.bzl b/third_party/google/protobuf/protobuf.bzl
index 23380ba..c08cd35 100644
--- a/third_party/google/protobuf/protobuf.bzl
+++ b/third_party/google/protobuf/protobuf.bzl
@@ -255,6 +255,7 @@ def cc_proto_library(
       hdrs=gen_hdrs,
       deps=cc_libs + deps,
       includes=includes,
+      copts=["-Wno-unused-parameter"],
       **kargs)
 
 def internal_gen_well_known_protos_java(srcs):
diff --git a/third_party/google/protobuf/util/python/BUILD b/third_party/google/protobuf/util/python/BUILD
index 3ac0385..7450dd2 100644
--- a/third_party/google/protobuf/util/python/BUILD
+++ b/third_party/google/protobuf/util/python/BUILD
@@ -1,3 +1,5 @@
+licenses(['permissive'])
+
 # This is a placeholder for python headers. Projects needing to use
 # fast cpp protos in protobuf's python interface should build with
 # --define=use_fast_cpp_protos=true, and in addition, provide
